docs/install/index.md

```diff
@@ -39,6 +39,91 @@ The database server can also be run on a separate server:
 
 Relationships between components on the same host are left out for brevity (except those making use of third-party components).
 
+### Requirements
+
+You need at least one machine on which to deploy the Opal PIE.
+This machine needs to have the [Docker Engine](https://docs.docker.com/engine/) and [Docker Compose](https://docs.docker.com/compose/) installed.
+A compatible container engine that also has support for compose can also be used.
+
+In addition, you need a Firebase project.
+
+#### Create and set up a new Firebase project
+
+If you don't have a dedicated Firebase project yet, follow the steps to [create a Firebase project](../development/local-dev-setup.md#create-a-new-firebase-project).
+If you already have a dedicated Firebase project, ensure that you have done the following steps:
+
+- [create a Realtime Database](../development/local-dev-setup.md#create-a-new-realtime-database)
+- [enable email and password authentication](../development/local-dev-setup.md#enable-email-and-password-authentication)
+- [retrieve Firebase configurations](../development/local-dev-setup.md#retrieve-the-firebase-project-configurations)
+
+#### Restrict service account permissions
+
+By default, Firebase creates a service account and API keys.
+The service account likely has more permissions than are needed.
+We recommend to restrict the permissions as much as possible.
+
+Go to the [Service Accounts in Google Cloud](https://console.cloud.google.com/projectselector2/iam-admin/serviceaccounts) and select your Firebase project, then:
+
+1. Select your Firebase project
```

Comment:
nit: It's asking twice to select your Firebase project, since it's also in the sentence before the list.